src/App.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState, useEffect } from 'react';
+import styles from './App.module.css';
+import API from './constant/api';
+import FLIGHTLISTS from './components/Flightlists/Flightlists';
+import Layout from './components/Layout/Layout';
+import SIDEFILTER from './components/Sidefilter/Sidefilter';
+import axios from 'axios';
+import moment from 'moment';
+
+const APP = () => {
+  const [flightData, flightDataChange] = useState([]);
+  const [filteredData, filteredDataChange] = useState([]);
+  const [returnData, returnDataChange] = useState([]);
+  const [isLoading, isLoadingChange] = useState(false);
+  const [showReturnData, showReturnDataHandler] = useState(false);
+  const [places, placesChange] = useState([]);
+  const [originCityInput, originCityInputHandler] = useState('');
+  const [destinationcityInput, destinationcityInputHandler] = useState('');
+
```

Comment:
callback function names should be setFlightData, setReturnData, setIsloading, setshowReturnData, setPlaces, setOriginCity, setDestinationCity